### PR TITLE
Feature/DGI-50: Add RazorEmail project and welcome email

### PR DESCRIPTION
**Pull request by ntw-ITM**
The large changes to the .sln file were probably made by Visual Studio when I added the new project. I don't think It's related to line endings.

### CODE REVIEW

File: backend/RazorEmails/Views/Shared/EmailButton.cshtml [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+  <!--[if mso]></center></v:textbox></v:roundrect></td></tr></table><![endif]-->
```
Comment (objmj-itminds) [inline backend/RazorEmails/Views/Shared/EmailButton.cshtml:6]: Please line end some more.

File: backend/RazorEmails/Views/Shared/EmailLayout.cshtml [hunk 2]
```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional //EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (objmj-itminds) [inline backend/RazorEmails/Views/Shared/EmailLayout.cshtml:1]: There is a lot of mixes between having a stylesheet and then inline style. Please pick a lane and stick to it.

Comment (objmj-itminds) [reply backend/RazorEmails/Views/Shared/EmailLayout.cshtml:1]: Make sure you are also following this
https://templates.mailchimp.com/getting-started/html-email-basics/

**Review by objmj-itminds — CHANGES_REQUESTED** [review]

**Comment by ntw-ITM** [comment]
Now I made a totally new email layout, using both the link you shared and this guide: https://webdesign.tutsplus.com/articles/creating-a-simple-responsive-html-email--webdesign-12978
It's a whole lot simpler than the generated one. It still uses a mix of both stylesheet and inline styling, but I hope it's not too much of an issue. I've made a small styling of a "mailSection" class in the stylesheet to be used by sections that are put into the body.

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**